### DISCUSSION TITLE
aside in title is not set aside

### DISCUSSION THREAD

**Discussion by JosiahParry** [quarto-dev/quarto-cli#3708] [Q&A]
When using a title slide in revealjs with a line under it and an aside the aside is in the middle and very much not "aside". 

How can this be fixed? 

<img width="750" alt="image" src="https://user-images.githubusercontent.com/13534508/208440013-d1ecb752-1ffe-4cb9-a956-5f887006de5c.png">

```quarto
---
title: "title"
format: revealjs
---

# this a title slide
this is a sentence under it

:::{.aside}
here is a note about the title
:::
```

**Comment by harrylojames** [comment]
Changing slide level from 3 to 2 fixed this for me

**Reply by JosiahParry** [reply]
what does this mean? Do you have an example

**Comment by cderv** [comment]
Currently aside works on content slide and not title /section slide, so using `##` header, not h1

````markdown
---
title: "title"
format: revealjs
---

## this a title slide

this is a sentence under it

:::{.aside}
here is a note about the title
:::

````

I am guessing here you want to add an aside on a title /section slide, right ?

**Reply by JosiahParry** [reply]
That is correct

**Reply by cderv** [reply]
Ok. So currently it is note supported to set aside on those slides. I don't know how / if we can tweak the CSS on those considering how the center is implementing. I'll open an issue about that.

**Reply by JosiahParry** [reply]
Good to know! Thanks for the follow up

**Comment by mcanouil** [comment]
Closing as outdated and in favour of:
- https://github.com/quarto-dev/quarto-cli/issues/4185
